Makefile

```diff
@@ -142,7 +141,7 @@ install-schemas:
 	install -Dm644 "$(DEV_DATA_DIR)/org.guake.gschema.xml" "$(DESTDIR)$(SCHEMA_DIR)/"
 
 compile-shemas:
-	if [ $(COMPILE_SCHEMA) = 1 ]; then glib-compile-schemas $(DESTDIR)$(gsettingsschemadir); fi
+	glib-compile-schemas $(DESTDIR)$(gsettingsschemadir)
```

Comment:
Distro packages MUST NOT have a compiled schema as this is handled globally outside of any package. For the same reason that distro packages must not run `update-desktop-database` or `update-mime-database` or other such commands.

The traditional way to detect that this should not be done, is to only do it when DESTDIR is empty. But existing distro packages are relying on using `make install COMPILE_SCHEMAS=0` which is opt in and anyone using that is definitely not going to be surprised that they need to compile the schemas themselves.

Choose either solution, but please restore the ability for distros to package Guake. :)

Reply:
Oh whoops, thought that was just redundant, opening new pull request with restoration

Reply:
Thanks! I do understand that maintaining compatibility with distro installs is not always the most obvious or intuitive, so sometimes things slip through the cracks... :)

Whole you are at it you can go the DESTDIR route, even. ;) It means distros don't need to pass two options when one option will do... Also, it will be more obvious to the next person working on the makefile, that this is special because of distro staged installs rather than just a random option.

Reply:
Oh, I put in a commit that just reverted, can look at an existence check on DESTDIR